artiq/coredevice/phaser.py

```diff
@@ -40,6 +40,16 @@
 PHASER_ADDR_DAC1_DATA = 0x28
 PHASER_ADDR_DAC1_TEST = 0x2c
 
+# servo registers
+PHASER_ADDR_SERVO_CFG0 = 0x30
+PHASER_ADDR_SERVO_CFG1 = 0x31
+
+# 0x32 - 0x61 ab regs
```

Comment:
Still `ab` and `AB_BASE`. It's both wrong (the order is `ba`) and unclear. `ba` or just call them `coefficients`.